packages/Algernon.package/ALGDynamicWorldMenuType.class/methodProperties.json

```diff
@@ -1,8 +1,6 @@
 {
 	"class" : {
-		"excludeDynamicWindows:" : "b 5/22/2021 20:46",
 		"extractMenuEntriesFrom:" : "NL 6/29/2021 15:00",
-		"filterOutDuplicates:" : "NM 6/29/2021 17:28",
-		"menuEntries:" : "NL 6/4/2021 16:21" },
+		"filterOutDuplicates:" : "NM 6/29/2021 17:28" },
```

Comment:
Really don't know how these changes got in here. As they defintively belong to prior merged stuff, maybe this was not merged correctly? 🤔
But it didn't breakt sth here, soo... 😅